packages/mobile/src/send/SendAmount.tsx

```diff
@@ -168,8 +171,12 @@ function SendAmount(props: Props) {
   const isAmountValid = parsedLocalAmount.isGreaterThanOrEqualTo(DOLLAR_TRANSACTION_MIN_AMOUNT)
   const isDollarBalanceSufficient = isAmountValid && newAccountBalance.isGreaterThanOrEqualTo(0)
 
-  const reviewBtnDisabled =
-    !isAmountValid || recipientVerificationStatus === RecipientVerificationStatus.UNKNOWN
+  const reviewButtonText =
```

Comment:
nit: was confused by the loading indicator being in a variable called text

Reply:
Renamed to `reviewButtonInnerElement`